ui/lib/pki/addon/routes/issuers/index.js

```diff
@@ -6,15 +6,23 @@ export default class PkiIssuersIndexRoute extends Route {
   @service secretMountPath;
   @service pathHelp;
 
+  beforeModel() {
+    // Must call this promise before the model hook otherwise it doesn't add OpenApi to record.
+    return this.pathHelp.getNewModel('pki/pki-issuer-engine', 'pki');
+  }
+
   model() {
     // the pathHelp service is needed for adding openAPI to the model
     this.pathHelp.getNewModel('pki/pki-issuer-engine', 'pki');
 
     return this.store
       .query('pki/pki-issuer-engine', { backend: this.secretMountPath.currentPath })
+      .then((issuersModel) => {
+        return { issuersModel, parentModel: this.modelFor('issuers') };
+      })
       .catch((err) => {
         if (err.httpStatus === 404) {
-          return [];
+          return { parentModel: this.modelFor('issuers') };
```

Comment:
needed for the EmptyState. This model returns information not on this model but for the parent model that is needed for links if any exist in the emptyState. (The empty states are conditional. The language and links change if the pki engine has been mounted and that part of the work has not been done yet).